sql/updates/world/master/2020_07_07_00_world.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` = 109132;
+INSERT INTO `world`.`spell_script_names` (`spell_id`, `ScriptName`) VALUES ('109132', 'spell_monk_roll');
```

Comment:
This query would never apply to a database named anything else than "world" - please adjust this by removing it.

Reply:
No, because it is in world folder and is correct. Sry i dont change